qa/suites/fs/mirror-ha/workloads/cephfs-mirror-ha-workunit.yaml

```diff
@@ -8,8 +8,13 @@ overrides:
         debug client: 10
 
 tasks:
+  - full_sequential_finally:
+    - exec:
+        mon.a:
+          - "ceph fs volume rm dc --yes-i-really-mean-it"
```

Comment:
Hmmm.. maybe I'm not seeing it - not cleaning up volumes was causing the an MDS failover?

Reply:
The MDS were being killed during teuthology unwind so some file systems had no MDS, causing the replacement warnings from the mons.

Reply:
I've added a comment about this.

Reply:
ah, of course +1